content/reference/core-configuration.md

```diff
@@ -0,0 +1,86 @@
+|---------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `serverUrl`         | The URL of the OpenID Connect server. <br />(Default: `http://localhost:8090`)    |                                                                                                                    
+| `realm`             | Identifies the authentication domain for which the authentication request is being made.                                                                              |
+| `client_id`         | The unique identifier assigned to the client application by the OIDC server.                                                                                      |
+| `client_secret`     | Used to authenticate the client when making confidential requests.                                                                       |
+| `username`          | The username credentials of the user who is attempting to authenticate with the OIDC server.                                                           |
+| `password`          | The password credentials of the user who is attempting to authenticate with the OIDC server.                                                                         |
+
+## OpenTelemetry
+
+| Attributes          | Description                                                                                                                                                                                    |
+|---------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `serverUrl`         | The URL of the OpenTelemetry server. <br />(Default: `localhost:4317`)                                                                                                                         | 
+| `samplingRate`      | The sampling rate for for traces. <br />(Default: `1`)                                                                                                                                         | 
+
+## SECRET
+
+| Attributes          | Description                                                                                                                                                                                    |
+|---------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `KEY`               | The SECRET KEY used for authorization within Core.       |                                                                                                                                                 
```

Comment:
Why is secret key in caps?

Reply:
The secret key config in Core is written like that in the config file, so we wanted to replicated its syntax exactly as seen.